ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -2658,6 +2658,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-zones-multisubnets-external-lb-usermanaged-f7
```

Comment:
`f7` sounds a bit aggressive, vsphere does not have a big number of slice quota, I think we can set it to `f28` to keep consistent with the above `vsphere-ipi-zones-multisubnets-external-lb` job.

And generally we always add a destructive ci job using the same install workflow as a pair.

Reply:
ok, updated f28